### PR TITLE
Prepare Microsoft.Azure.WebJobs.Extensions.EventGrid 3.0.0-beta.1 release

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 3.0.0-beta.1 (Unreleased)
+## 3.0.0-beta.1 (2021-04-06)
```
Comment (JoshLove-msft) [inline sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/CHANGELOG.md:3]: Aren't you releasing today?

File: sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Microsoft.Azure.WebJobs.Extensions.EventGrid.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Description>This extension adds bindings for EventGrid</Description>
```
Comment (JoshLove-msft) [inline sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Microsoft.Azure.WebJobs.Extensions.EventGrid.csproj:4]: Use package ref?

File: sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/CHANGELOG.md (hunk 1 above)
Comment (pakrym) [reply sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/CHANGELOG.md:3]: That's the script :) Will fix.

File: sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Microsoft.Azure.WebJobs.Extensions.EventGrid.csproj (hunk 2 above)
Comment (pakrym) [reply sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Microsoft.Azure.WebJobs.Extensions.EventGrid.csproj:4]: Yep, that's why it's back to draft :)

**Comment by pakrym** [comment]
Updated.

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**